src/lib/ui/Settings.svelte

```diff
@@ -48,7 +62,17 @@
         <p class="font-medium mb-2">Color palette</p>
         <p class="text-sm text-zinc-400">Color scheme for text in terminals.</p>
       </div>
-      <div class="text-red-500">Coming soon</div>
+      <div>
+        <select
+          class="w-52 px-3 py-1.5 rounded-md bg-zinc-700 outline-none focus:ring-2 focus:ring-indigo-500"
```

Comment:
From your screenshot, the arrow at the right side of the dropdown (when closed) is a bit too close to the right edge of the element